shared/src/config.ts

```diff
@@ -0,0 +1,29 @@
+type SharedConfigT = {
+    cms: boolean | undefined;
+};
+
+class Configuration {
+    private config: SharedConfigT;
+
+    constructor(conf: SharedConfigT) {
+        this.config = conf;
+    }
+
+    public setConfig(newConfig: SharedConfigT) {
+        this.config = newConfig;
+    }
+
+    public getConfig() {
+        return this.config;
+    }
+
+    public getCmsFlag() {
+        return this.config.cms;
+    }
+}
+
+export const config: Configuration = new Configuration({ cms: true });
```

Comment:
What I was saying in the standup, how I expected this to work is this class is only instantiated the first time this variable is imported. Evidently this is not the case as `config` is definitely imported before `db`, but the latter already exists. So a good learning opportunity to dive a bit into how JS/TS does these things.

Reply:
![Sequence](https://github.com/user-attachments/assets/e70c98e3-6def-4c38-ba3c-4831e390cb84)
From this image, you can see that the area where the config is made in cms, is run after the database. So I agree, Something worth going below the water for. Going to research this and how NodeJS handles modules and export variables, before continuing on this PR.